# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sgutils/include.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- sg3_utils-1.26/include/sg_cmds_basic.h.vanilla	2008-06-13 09:30:26.000000000 +0200
+++ sg3_utils-1.26/include/sg_cmds_basic.h	2008-06-13 09:30:40.000000000 +0200
@@ -30,6 +30,8 @@
  *
  */
 
+#include <inttypes.h>
+
 #ifdef __cplusplus
 extern "C" {
 #endif
